### PR TITLE
feat(owl-bot): create issues if it finds malformed config

### LINKED ISSUE #1760: owl bot: scan-configs command should report a bug if it finds a mal-formed config file.
scan-configs scans all the repos in googleapis, searching for all the .OwlBot.yaml and .OwlBot.lock.yaml files.  If a single one of those configs is malformed, it logs an error message, but doesn't notify the repo owner in any way.

scan-configs should report a bug and continue to the next repo if it finds a mal-formed config.

### PR DESCRIPTION
**Pull request by danielbankhead**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [x] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/repo-automation-bots/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

Fixes #1760 🦕

### CODE REVIEW

File: packages/owl-bot/test/handlers.ts [hunk 1]
```diff
@@ -404,14 +416,39 @@ describe('refreshConfigs', () => {
...
     assert.deepStrictEqual(configsStore.configs, new Map());
   });
+
+  it('creates issues when configs cannot be loaded', async () => {
+    const configsStore = new FakeConfigsStore();
+    const universalInvalidContent = 'deep-copy-regex\n - invalid_prop: 1';
+
+    sandbox.stub(core, 'getFileContent').resolves(universalInvalidContent);
+
+    const a = sandbox.spy(octokitSha123.issues, 'create');
```
Comment (tmatsuo) [inline packages/owl-bot/test/handlers.ts:426]: `a` seems like too short a name. Can you make it something like `issuesCreateSpy`?

**Review by tmatsuo — CHANGES_REQUESTED** [review]

**Comment by tmatsuo** [comment]
Also, before deploying, we should make sure owl bot installations have a read/write permission on issues.

File: packages/owl-bot/src/octokit-util.ts [hunk 2]
```diff
@@ -75,3 +76,44 @@ export function octokitFactoryFrom(params: OctokitParams): OctokitFactory {
...
+  // check if issue exists
+  const issues = await octokit.issues.listForRepo({
+    owner,
+    repo,
+    per_page: 100,
+    state: 'open',
+  });
+
+  for (const issue of issues.data) {
+    if (issue.title === title) {
```
[the remark this answers is not in the document]
Comment (danielbankhead) [reply packages/owl-bot/src/octokit-util.ts:103]: The issues returned here are open (see line 99). Do you think it's worth reopening an issue or creating a new one?

[the remark this answers is not in the document]
Comment (tmatsuo) [reply packages/owl-bot/src/octokit-util.ts:103]: Ah nice I didn't noticed the filter, then LGTM.

**Review by tmatsuo — APPROVED** [review]
I'd wait for @SurferJeffAtGoogle approve this too.

**Comment by danielbankhead** [comment]
Thanks @tmatsuo, @SurferJeffAtGoogle and I paired on the implementation portion of this while I wrote the tests. I'll verify `owl-bot` has a read/write permission on issues.

File: packages/owl-bot/src/handlers.ts [hunk 3]
```diff
@@ -271,6 +271,20 @@ export async function refreshConfigs(
       logger.error(
         `${repoFull} has an invalid ${owlBotLockPath} file: ${e.message}`
       );
+
+      const title = `Invalid ${owlBotLockPath}`;
+      const body = `\`owl-bot\` will not be able to update this repo until '${owlBotLockPath}' is fixed.
+
+Please fix this as soon as possible so that your repository will not go stale.`;
+
+      await createIssueIfTitleDoesntExist(
```
Comment (bcoe) [inline packages/owl-bot/src/handlers.ts:280]: I love method names like this 😄

**Review by bcoe — APPROVED** [review]
This looks like a very clean implementation, and has great tests 👍 

We should be a little bit mindful when we role this out, we can manually run the scan config command, and then run a command to check if there are any existing dubious configs.

The thing we should keep an eye on is too many issues being opened 😄

**Pull request merged**